update(docs): update installation documentation: cloning, composer setup, and database config

labmecanicatec opened the pull request:
This update improves the installation instructions by:

Adding steps to clone the application from the official GitHub repository.

Including instructions to install PHP dependencies using Composer.

Adding setup guidance for the config.php file.

Documenting required database configuration fields ($conf['settings']['database']).

Noting that the database user must have privileges to create the database and tables.

Requiring configuration of $conf['settings']['install.password'] before running the installer.


These changes aim to provide clearer guidance for first-time users and ensure a smoother setup process.

Closes #595.

JohnVillalovos commented:
> Can anyone help me check the appearance? I can't use the preview in INSTALLATION.rst.

I see it here: https://github.com/labmecanicatec/app/blob/docs/update-installation-guide/docs/source/INSTALLATION.rst

The other thing is that there will be a readthedocs build on each PR. I enabled that earlier today.

![image](https://github.com/user-attachments/assets/0950f4ca-08cf-4b8c-8912-6bf96b42d212)

https://librebooking--675.org.readthedocs.build/en/675/INSTALLATION.html

JohnVillalovos commented on docs/source/INSTALLATION.rst:
```diff
@@ -100,8 +127,15 @@ You must have the application configured correctly before running the
+
+This password is required to access the installer.
+
+Then, navigate to the ``/Web/install`` directory in a web browser and follow the on-screen instructions.
```
Need a blank line after this line. To make sphinx happy. Or prefix the line with `|`

https://docutils.sourceforge.io/docs/user/rst/quickref.html#line-blocks

labmecanicatec replied:
In other words, each paragraph should start with "|"? So I have to better condition my work environment to be able to manupillary the .rst files well

JohnVillalovos replied:
So the `|` is the equivalent in Markdown when a line has two trailing space characters.

https://www.markdownguide.org/basic-syntax/#line-break-best-practices

I went ahead and changed it to not use that. and use a `.. note::` section

labmecanicatec commented:
> The commit message should start with `docs:` to make comittizen happy.
> 
> `docs: update installation documentation: cloning, composer setup, and database config`

I'm having authentication issues with git push --force, can you help me with that?

labmecanicatec committed:
docs: update installation docs cloning, composer setup, and database config

This update improves the installation instructions by:

  * Adding steps to clone the application from the official GitHub
    repository.

  * Including instructions to install PHP dependencies using Composer.

  * Adding setup guidance for the config.php file.

  * Documenting required database configuration fields
    ($conf['settings']['database']).

  * Noting that the database user must have privileges to create the
    database and tables.

  * Requiring configuration of $conf['settings']['install.password']
    before running the installer.

  * Clarifying that the installer can be accessed via /Web/install in
    the browser.

  * These changes aim to provide clearer guidance for first-time users
    and ensure a smoother setup process.

Closes #595.

JohnVillalovos reviewed (approved):
This looks great @labmecanicatec 

Thank you very much for doing this.
